layout/_third-party/analytics/lean-analytics.swig

```diff
@@ -68,25 +68,29 @@
               }
             });
           } else {
-            var newcounter = new Counter();
-            /* Set ACL */
-            var acl = new AV.ACL();
-            acl.setPublicReadAccess(true);
-            acl.setPublicWriteAccess(true);
-            newcounter.setACL(acl);
-            /* End Set ACL */
-            newcounter.set("title", title);
-            newcounter.set("url", url);
-            newcounter.set("time", 1);
-            newcounter.save(null, {
-              success: function(newcounter) {
-                var $element = $(document.getElementById(url));
-                $element.find('.leancloud-visitors-count').text(newcounter.get('time'));
-              },
-              error: function(newcounter, error) {
-                console.log('Failed to create');
-              }
-            });
+            {% if theme.leancloud_visitors.security %}
+              console.log('Counter not initialized!');
```

Comment:
Ok, i see what here is `security` option just add warning in console log. There is some my 5 cents on it:

1. Replace `console.log` with `hexo.log.warn` for better warning message.
2. `security` can be set to `true` by default. So, if user wan't to install security plugin, he can turn off `security` option. If he don't know about security (newbie in NexT) and just turn on LC option, he will seen this warning message with following instructions to install plugin.
3. Warning message shoukd be informative. Instead of `Counter not initialized!` something like `ATTENTION! LeanCloud counter have security bug, for solve it see here: https://github.com/theme-next/hexo-leancloud-counter-security`.

OR

We can just cut out `security` option and add in NexT config comments under leancloud options what there are some security issues and can be solved under the plugin link. For now option — it's just console warning, no more. Who know about this — will know, option will be useless, i think. 1 excess interation in swig.

Reply:
but this file is excuted in browser where there is no hexo.log.warn

Reply:
Oh! So, why this warn needed if user will not see it?

Reply:
because if they turn on the `security` but not install the plugin and follow the instruction to config th e Leancloud background, the counter will not work at all

Reply:
So, stay as it? False by default?

***

Also, need to provide some badges for [this repo](https://github.com/theme-next/hexo-leancloud-counter-security). [NPM link](https://www.npmjs.com/package/hexo-leancloud-counter-security), version, compatability, etc.